db/migrate/20200206151814_create_pet_applications.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePetApplications < ActiveRecord::Migration[5.1]
+  def change
+    create_table :pet_applications do |t|
+      t.references :pet, foreign_key: true
+      t.references :application, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
This is our join table for the many to many relationship between Applications and Pets.